config/crd/bases/operator.containers.carbonblack.io_cbcontainersagents.yaml

```diff
@@ -3439,19 +3439,19 @@ spec:
                           cliFlags:
                             default: {}
                             properties:
-                              ignoreBuiltInRegex:
+                              enableSecretDetection:
```

Comment:
update the CRD under charts as well (needs to be manual :( )

Reply:
In which file?